src/server.ts

```diff
@@ -64,6 +65,35 @@ newApp(
       });
     }
 
+    // Monitor all the requests to the server and track all to AppInsights
+    if (process.env.APPINSIGHTS_INSTRUMENTATIONKEY !== undefined) {
```

Comment:
I believe the appinsights sdk provides automatic collection of requests from node see [docs](https://github.com/microsoft/ApplicationInsights-node.js#configuration) - also you may need to flush the buffer on shutdown, see [here](https://github.com/microsoft/ApplicationInsights-node.js/blob/50dec0941d5e73de6b7ffe081f66ab739bd62876/Tests/FunctionalTests/TestApp/Main.js#L71)

Reply:
fixed in 41c9e35